src/test/java/net/rithms/test/UtilTest.java

```diff
@@ -68,4 +68,21 @@ public void testJoinString() throws RiotApiException {
 		thrown.expect(NullPointerException.class);
 		RiotApiUtil.joinString(null, (CharSequence[]) null);
 	}
+	
+	@Test
+	public void testRequireValidSummonerName() {
+		assertEquals("_Fiddlesticks.123", RiotApiUtil.requireValidSummonerName("_Fiddlesticks.123"));
+		assertEquals("日本語を話しますか", RiotApiUtil.requireValidSummonerName("日本語を話しますか"));

```

Comment:
It might be beneficial to make a comment about each language we are testing for, as these language symbols may not be immediately obvious to all readers.